client/src/app/users/user.component.html

```diff
@@ -1,7 +1,11 @@
-<div *ngIf="user" class="user">
-  <ul>
-    <li>Name: {{ user.name }}</li>
-    <li>Age: {{ user.age }}</li>
-    <li>Company: {{ user.company }}</li>
-  </ul>
-</div>
+<mat-expansion-panel *ngIf="user" class="user" matTooltip="email: {{user.email}}" #elem [id]="user.email">
```

Comment:
the `*ngIf` here is also probably unnecessary. The class should probably be something like `user-panel`. 

We shouldn't use tooltips like this. That is not what they are for.

Reply:
Good point about tooltips not really being for this kind of thing. We should consider displaying that information (email) in some other way than what it has been for the past few years so that we are modeling good use of tooltips. Are you saying instead of user to call this `user-panel`? Maybe we could also have a `user-card` that would be a thing that could be in some other view of the user data... I agree that `user-panel` may make more sense since it's part of an accordion. Is there a way for us to better highlight the use of nesting components?

Reply:
I have renamed the class to "user-panel", but what does it mean by highlighting the nesting component? 

We can display the email together with other user info. Where can we add the tooltips?